test/extended/testdata/jenkins-slave-template.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: jenkins-slave-pipeline
+objects:
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    creationTimestamp: null
+    name: openshift-jee-sample
+  spec:
+    strategy:
+      jenkinsPipelineStrategy:
+        jenkinsfile: |-
+          try {
+             timeout(time: 20, unit: 'MINUTES') {
+        
+                node("jenkins-slave") {
+                  stage("Checkout") {
+                    git url: "https://github.com/openshift/openshift-jee-sample.git", branch: "master"
+                  }
+                  stage("Build WAR") {
+                    sh "mvn clean package -Popenshift"
+                  }
+                }
```

Comment:
let's make this job simpler so it runs faster.  Something minimal that just confirms we're in the right slave image and runs quickly (like maven version or something where we confirm the command doesn't fail (no point checking the actual version value since that will change over time))